install.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh 
```

Comment:
We're not sure this install.sh script is the best way to do the path setup. We think it would either be better to have this script append to your .bashrc so that it doesn't have to be run every time, OR avoid environment variables entirely (the mappot_fitacf script could potentially use relative paths to the key or extension files, or it could source a file that specified where to place the plots, etc.)